app/views/publisher/_publisher.html.haml

```diff
@@ -57,7 +57,8 @@
           .btn-toolbar.pull-right#publisher-service-icons
             - if current_user.services
```

Comment:
Sidenote, as you didn't change it here, but I don't think this `if` is needed, as if there are no services, the list would just be empty, so there is nothing to loop over anyway?